src/components/Dissolution/CustodianOfRecords.vue

```diff
@@ -187,21 +187,21 @@
     <template v-else>
       <div class="summary-section">
         <v-row no-gutters>
-          <v-col cols="12" md="5" lg="5">
+          <v-col cols="12" md="5" lg="5" class="pr-8">
             <div class="summary-sub-label">Name</div>
             <span class="summary-text">{{ getCustodianName }}</span>
           </v-col>
-          <v-col cols="12" md="5" lg="5" class="pl-8">
```

Comment:
This caused this column to be indented when it wraps to its own line. Instead, I added right padding to the first column.